docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -171,8 +171,16 @@ TBD.
 tests directory
 ----------------
 
-TBD. Expect tests for the collection itself to reside here.
+Executing sanity, integration, and unit tests is largely identical to performing these tests within Ansible's mainline code. ``ansible-test``'s targets must execute from a directory path with the following format: ``collections/<namespace>/<collection>``. For example, ``ansible-test`` may be executed against IOS modules from the default collection location of ``~/.ansible/collections/cisco/ios/``. Tests may also be executed while in a custom directory, such as ``~/work/ansible/collections/cisco/ios/``, as long as it is the current path.
```

Comment:
Recommend re-phrasing the second sentence here for clarity: ``ansbile-test`` executes against modules implemented from the relative path ``collections/<namespace>/<collections>.

Reply:
I updated. What do you think now?